tests/pos/not-looping-implicit.scala

```diff
@@ -24,10 +24,12 @@ object Schema {
     inline summonInline[Mirror.Of[A]] match {
       case m: Mirror.SumOf[A]     =>
         lazy val members     = recurse[m.MirroredElemLabels, m.MirroredElemTypes]()
-        new Schema[A] {}
+        class InlinedSchema extends Schema[A] {} // avoid inline anon class warning
+        new InlinedSchema
```

Comment:
It doesn't change anything in the test, but it's better for lisibility
```suggestion
        ???
```